vector/src/main/res/values-fr/strings.xml

```diff
@@ -863,6 +863,17 @@ Appareils inconnus :</string>
     <string name="group_details_rooms">Discussions</string>
     <string name="no_users_placeholder">Pas d\'utilisateurs</string>
 
+    <!-- Login/Register Tchap -->
+    <string name="tchap_register_title">Inscription Tchap</string>
+    <string name="tchap_connection_title">Connexion Tchap</string>
+    <string name="tchap_has_account_no">Je n’ai pas\nde compte</string>
+    <string name="tchap_has_account_yes">J’ai un compte</string>
+    <string name="tchap_connection_email">Adresse mél</string>
```

Comment:
Please replace "mél" with "email" (in each occurrences)

Reply:
done